recipe/session/accessToken.go

```diff
@@ -32,49 +33,32 @@ type accessTokenInfoStruct struct {
 	timeCreated             uint64
 }
 
-func getInfoFromAccessToken(token string, jwtSigningPublicKey string, doAntiCsrfCheck bool) (*accessTokenInfoStruct, error) {
-	payload, err := verifyJWTAndGetPayload(token, jwtSigningPublicKey)
+func getInfoFromAccessToken(jwtInfo ParsedJWTInfo, jwtSigningPublicKey string, doAntiCsrfCheck bool) (*accessTokenInfoStruct, error) {
+
+	err := verifyJWT(jwtInfo, jwtSigningPublicKey)
 	if err != nil {
-		return nil, errors.TryRefreshTokenError{
-			Msg: err.Error(),
-		}
+		return nil, err
```

Comment:
you need to return try refresh token.. not sure why you removed the code for that..